src/results/electric_tariff.jl

```diff
@@ -14,6 +14,22 @@
 - `year_one_export_benefit_before_tax` export credits over the first year, before considering tax benefits. A positive value indicates a benefit. 
 - `lifecycle_coincident_peak_cost_after_tax` lifecycle coincident peak charge in present value, after tax
 - `year_one_coincident_peak_cost_before_tax` coincident peak charge over the first year
+
+- `energy_charge_cost_matrix` matrix for cost of electricity (rows = cost for each timestep, columns = # of tiers)
+- `facility_demand_charge_cost_matrix` matrix for facility demand charges (rows = months, columns = # of tiers)
+- `utility_to_gross_loads_cost_series` timeseries of cost of gross electricity purchases (grid to load, grid to battery)
+- `monthly_facility_demand_cost_series` monthly facility demand charges
+- `utility_to_storage_cost_series` the cost of energy purchased from the grid to charge batteries in each timestep
+- `monthly_utility_to_storage_cost_series` monthly totals of `utility_to_storage_cost_series`
+- `monthly_utility_to_gross_loads_cost_series` monthly totals of `utility_to_gross_loads_cost_series`
+- `tou_demand_cost_series` create list with month, TOU rate, peak demand for that TOU period, TOU demand charge for each TOU time period. Tiered TOU demand charges are also factored in.
```

Comment:
I'd like to avoid the string concatenation approach and instead use the matrix->nested array or dictionary approach for this that we chatted about for this. Let me know what you think would be best out of those two options. The nested array seems more consistent with the other matrix outputs, but the dictionary might be more understandable without documentation next to you.

Reply:
Updated, please check out the outputs.